public-api/src/integration/java/org/pmiops/workbench/publicapi/DataBrowserControllerIntegrationTest.java

```diff
@@ -32,17 +34,27 @@
 public class DataBrowserControllerIntegrationTest {
 
   public static String DB_API_BASE_PATH = "DB_API_BASE_PATH";
+  public static String CLIENT_ID = "238501349883-965gu9qminos5dfcpusi43eokvd5i3io.apps.googleusercontent.com";
```

Comment:
This is only the staging client ID right? If so, should clarify this, or else make this an environment variable provided to the test.

Reply:
Added a comment on top.